snitch/regs.py

```diff
@@ -61,11 +61,11 @@ def __init__(
             stream_def = StreamDef()
             self.dm_by_id.append(stream_def)
             self.streams[reg] = stream_def
-        super().__init__(infinite_regs)
+        super().__init__(infinite_regs, flen=32)
```

Comment:
Should we use f64 by default?

Reply:
The current version of frep does not actually support 64 bits as of yet. Should be a straightforward change though

Reply:
I think Snitch only supports flen=64 so would be good for this to be parametrizeable

Reply:
I'll start working on a patch that makes this work, and then see if we can reproduce some numbers from docker!